packages/components/tests/acceptance/percy-test.js

```diff
@@ -104,6 +104,12 @@ module('Acceptance | Percy test', function (hooks) {
     await visit('/components/side-nav');
     await percySnapshot('SideNav');
 
+    await visit('/components/segmented-group');
```

Comment:
can we add this in a separate PR so it can get in while separator is being worked on?

Reply:
Makes sense – raised https://github.com/hashicorp/design-system/pull/1397 to keep them separated